<?xml version="1.0"?>
<launch>
    <node pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager" />
    <node pkg="nodelet" type="nodelet" name="image_nodelet" args="load depth_image_proc/point_cloud_xyzrgb nodelet_manager">
        <remap from="depth_registered/camera_info" to="camera/front/depth/camera_info" />
        <remap from="depth_registered/image_rect" to="camera/front/depth/image_raw" />
        <remap from="rgb/camera_info" to="/camera/front/aligned_depth_to_color/camera_info" />
        <remap from="rgb/image_rect_color" to="/camera/front/aligned_depth_to_color/image_raw" />
        <remap from="depth_registered/points" to="camera/front/depth/color/points" />
        <param name="queue_size" value="2" />
    </node>
</launch>
